seed/seedwriter/writer.go

```diff
@@ -481,6 +491,11 @@ func (w *Writer) Start(db asserts.RODatabase, f SeedAssertionFetcher) error {
 		}
 	}
 
+	// fetch model validation sets if any
+	if err := w.fetchValidationSets(f); err != nil {
```

Comment:
Just curious why we do not call this `fetchModelValidationSets` based on your comment?